### PR TITLE
Media functions compatible with sites 7.x sites that have PropertyValueConverters enabled

### PR DESCRIPTION
**Pull request by tomvanenckevort-hp**
Still backwards compatible with sites that don't have them enabled.

### CODE REVIEW

File: Endzone.Umbraco.Extensions/PublishedContentExtensions/Media.cs [hunk 1]
```diff
@@ -21,21 +22,40 @@ public static IEnumerable<IPublishedContent> GetMultipleTypedMedia(this IPublish
...
+            else if (contentValue is string)
+            {
+                // use old comma separated list
+                var imageIds = contentValue.ToString().Split(new[] { "," }, StringSplitOptions.RemoveEmptyEntries);
+                var umbracoHelper = new UmbracoHelper(UmbracoContext.Current);
+
+                return umbracoHelper.TypedMedia(imageIds);
+            }
+
+            return Enumerable.Empty<IPublishedContent>();
```
Comment (barryokane) [inline Endzone.Umbraco.Extensions/PublishedContentExtensions/Media.cs]: In what situation would this get returned?

Comment (tomvanenckevort-hp) [reply Endzone.Umbraco.Extensions/PublishedContentExtensions/Media.cs]: If it's none of the above, which could be the case if there is no value stored for the property (would return null instead).

Comment (barryokane) [reply Endzone.Umbraco.Extensions/PublishedContentExtensions/Media.cs]: Wouldn't `!content.HasValue` cover that?  Or am I misunderstanding?

Comment (tomvanenckevort-hp) [reply Endzone.Umbraco.Extensions/PublishedContentExtensions/Media.cs]: Good point. I could make the last `else if` an `else` and assume it's always a string.

Comment (barryokane) [reply Endzone.Umbraco.Extensions/PublishedContentExtensions/Media.cs]: No, its fine - I was just checking my understanding!  I like the check for a string as it makes that explicit.

Just to be pedantic : maybe add a comment to explain the last return (as I'll probably forget this next time I look at it!)

**Review by barryokane — APPROVED** [review]

**Pull request merged**